pkg/kudoctl/kudoinit/prereq/webhook.go

```diff
@@ -111,6 +111,10 @@ func (k *KudoWebHook) verifyWithCertManager(client *kube.Client, result *verifie
 	if err := k.validateCertManagerInstallation(client, result); err != nil {
 		return err
 	}
+	if !result.IsValid() {
```

Comment:
wait so verifier returns errors but you still have to check the result? 🤔 that's kind of error-prone...

Reply:
Yeah, it is... It's about different things:
- The return value determines if there was an error in the validation process
- The result contains errors that the validation detected

In hindsight it would be better to have just the result and handle the validation exceptions there as well...

Reply:
Yeah that sounds like better design probably...